test/e2e/framework/volume/fixtures.go

```diff
@@ -202,6 +202,23 @@ func NewGlusterfsServer(cs clientset.Interface, namespace string) (config TestCo
 	_, err := cs.CoreV1().Endpoints(namespace).Create(context.TODO(), endpoints, metav1.CreateOptions{})
 	framework.ExpectNoError(err, "failed to create endpoints for Gluster server")
 
+	service := &v1.Service{
```

Comment:
Should the test create Service first and then the Endpoint? Because the endpoint is without a service for a short time and could be garbage collected,

Reply:
done.